packages/components/src/config/variables.ts

```diff
@@ -31,6 +31,7 @@ export const LAYOUT_SIZE = {
     MENU_SECONDARY_WIDTH: '300px',
     GUIDE_PANEL_WIDTH: '350px',
     GUIDE_PANEL_CONTENT_WIDTH: '305px',
+    PIN_MATRIX_MAX_WIDTH: '316px',
```

Comment:
I don't think it's related to layout calculations. Why not have this defined in PinMatrix? :)

Reply:
done